config/environments/development.rb

```diff
@@ -51,14 +70,15 @@
   # Suppress logger output for asset requests.
   config.assets.quiet = true
 
-  # Raises error for missing translations
-  # config.action_view.raise_on_missing_translations = true
-  logger           = ActiveSupport::Logger.new(STDOUT)
-  logger.formatter = config.log_formatter
-  config.logger    = ActiveSupport::TaggedLogging.new(logger)
+  # Raises error for missing translations.
+  # config.i18n.raise_on_missing_translations = true
+
+  # Annotate rendered view with file names.
+  # config.action_view.annotate_rendered_view_with_filenames = true
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   # Commented out for development purposes with Apple M1 Silicon Procesing Chips
   # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  config.hosts << "oralhistory.test"
```

Comment:
Had an error in the browser once navigatting to oralhistory.test about this issue: https://blog.saeloun.com/2019/10/31/rails-6-adds-guard-against-dns-rebinding-attacks, https://guides.rubyonrails.org/configuring.html.

So I added the below code to config/environments/development.rb:
config.hosts << "oralhistory.test"

Reply:
this is expected